WooCommerce/Classes/POS/Presentation/Reusable Views/POSSendReceiptView.swift

```diff
@@ -86,6 +98,7 @@ struct POSSendReceiptView: View {
         }
         .padding([.horizontal, .bottom])
         .animation(.easeInOut, value: errorMessage)
+        .animation(.easeInOut, value: keyboardObserver.keyboardHeight)
```

Comment:
I'm not sure how nicely this animation will play if you do the separate viewmodifier though. It may still be possible, but check before you go too deep.